src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -32,37 +32,41 @@ function createModalStackNavigator(screens) {
 }
 
 // We use getComponent/require syntax so that file used by screens are not loaded until we need them.
-const IOUBillStackNavigator = createModalStackNavigator([
+const MoneyRequestModalStackNavigator = createModalStackNavigator([
     {
         getComponent: () => {
-            const IOUBillPage = require('../../../pages/iou/IOUBillPage').default;
-            return IOUBillPage;
+            const MoneyRequestAmountPage = require('../../../pages/iou/steps/MoneyRequestAmountPage').default;
+            return MoneyRequestAmountPage;
         },
-        name: 'IOU_Bill_Root',
+        name: 'Money_Request',
     },
     {
         getComponent: () => {
-            const IOUCurrencySelection = require('../../../pages/iou/IOUCurrencySelection').default;
-            return IOUCurrencySelection;
+            const MoneyRequestEditAmountPage = require('../../../pages/iou/steps/MoneyRequestAmountPage').default;
+            return MoneyRequestEditAmountPage;
         },
-        name: 'IOU_Bill_Currency',
+        name: 'Money_Request_Amount',
+    },
+    {
+        getComponent: () => {
+            const MoneyRequestParticipantsPage = require('../../../pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage').default;
```

Comment:
That is a mistake no?
```suggestion
            const MoneyRequestParticipantsPage = require('../../../pages/iou/steps/MoneyRequstParticipantsPage').default;
```

Reply:
Nope. That is how it's structured
<img width="433" alt="image" src="https://github.com/Expensify/App/assets/50919443/c2582709-05e2-49df-a7d7-0aa99e7d2303">

But I realized the folder name has a typo `Requst` 🤣. Should we fix it now? I'm afraid changing it will make the review hard.
<img width="446" alt="image" src="https://github.com/Expensify/App/assets/50919443/2da2bfac-2547-43b6-8a7c-646d293b5642">

Reply:
Ah that is confusing.

Lets fix it in a follow up